File: lib/src/main/java/com/fanwe/lib/http/utils/HttpLog.java
package com.fanwe.lib.http.utils;

import android.util.Log;

import com.fanwe.lib.http.RequestManager;

/**
 * Created by zhengjun on 2017/9/20.
 */
public class HttpLog
{
    public static final String TAG = "FHttp";

    public static boolean isDebug()
    {
        return RequestManager.getInstance().isDebug();
    }

    public static void i(String msg)
    {
        if (!isDebug())
        {
            return;
        }
        Log.i(TAG, msg);
    }

    public static void e(String msg)
    {
        if (!isDebug())
        {
            return;
        }
        Log.e(TAG, msg);
    }
}

File: lib/src/main/java/com/sd/lib/http/utils/HttpLog.kt
package com.sd.lib.http.utils

import android.util.Log
import com.sd.lib.http.RequestManager

internal class HttpLog private constructor() {

    companion object {
        private const val TAG = "FHttp"

        @JvmStatic
        fun i(msg: String?) {
            if (RequestManager.instance.isDebug) {
                Log.i(TAG, msg)
            }
        }

        @JvmStatic
        fun e(msg: String?) {
            if (RequestManager.instance.isDebug) {
                Log.e(TAG, msg)
            }
        }
    }
}